fiftyone/factory/repos/execution_store.py

```
@@ -800,6 +927,74 @@ def set_cache_key(
+        self,
+        store_name,
+        key,
+        value,
+        ttl=None,
+        policy="persist",
+    ):
+        composite_key = self._doc_key(store_name, key)
+        with self._lock:
+            if composite_key in self._docs:
+                return False
+            now = datetime.utcnow()
```

Ruff (0.16.2)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)